include/fem-dem/gls_vans.h

```diff
@@ -148,9 +148,65 @@ class GLSVANSSolver : public GLSNavierStokesSolver<dim>
   void
   read_dem();
 
+
+  /**
+   * @brief This function gives the periodic offset of the domain which is needed
```

Comment:
```suggestion
   * @brief This function returns the periodic offset of the domain which is needed
```

Reply:
As we discussed, "offset" can be a bunch of things and from my perspective, the purpose of it is not very clear. If you could bring here the explanation you gave me on slack, it would be more than perfect.